README.md

```diff
@@ -58,6 +58,74 @@ async fn main() {
 }
 ```
 
+## Running on Lambda
+
+The Restate Rust SDK supports running services on AWS Lambda using Lambda Function URLs. This allows you to deploy your Restate services as serverless functions.
+
+### Setup
+
+First, enable the `lambda` feature in your `Cargo.toml`:
+
+```toml
+[dependencies]
+restate-sdk = { version = "0.1", features = ["lambda"] }
+tokio = { version = "1", features = ["full"] }
+```
+
+### Basic Lambda Service
+
+Here's how to create a simple Lambda service:
+
+```rust
+use restate_sdk::prelude::*;
+
+#[restate_sdk::service]
+trait Greeter {
+    async fn greet(name: String) -> HandlerResult<String>;
+}
+
+struct GreeterImpl;
+
+impl Greeter for GreeterImpl {
+    async fn greet(&self, _: Context<'_>, name: String) -> HandlerResult<String> {
+        Ok(format!("Greetings {name}"))
+    }
+}
+
+#[tokio::main]
+async fn main() {
+    // To enable logging/tracing
+    // tracing_subscriber::fmt::init();
```

Comment:
Add a link to this page here https://docs.aws.amazon.com/lambda/latest/dg/rust-logging.html#rust-logging-tracing